content/docs/1.5/scalers/cron.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Cron"
+layout = "scaler"
+availability = "v1.5+"
+maintainer = "Community"
+description = "Scale applications based on a cron schedule."
+go_file = "cron_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `cron` trigger that scales based on a Cron Schedule.
+
+```yaml
+triggers:
+- type: cron
+  metadata:
+    # Required
+    timezone: Asia/Kolkata  # The acceptable values would be a value from the IANA Time Zone Database.
+    start: 0 0/30 * * *
```

Comment:
What kind of Cron sytax are we using here? Are we using standard of do we support some non-standard syntax? From the `/` it seems like that we support some non-standard one. https://en.wikipedia.org/wiki/Cron Would be nice to describe. 

What is this specific example specify? (hours = 0/30) ?

Reply:
I'm wondering if cron is actually the best approach for defining this meaning, why don't we use time stamps for defining start & end of a window (in UTC ofcourse)?

A bit late to come up with this, but was just wondering about the reasoning behind it.

Reply:
This is the library I used. https://github.com/ringtail/go-cron
0 0/30 * * * indicates "Every 30 minutes starting at :00 minute after the hour"

Reply:
The Slashes are non standard characters as specified in the wiki itself. 
Slash (/)
    In vixie-cron, slashes can be combined with ranges to specify step values. For example, */5 in the minutes field indicates every 5 minutes. So, '/' is used mostly to describe frequencies.

Reply:
@tomkerkhove I think that cron is a good approach to specify this kind of (repetitive) tasks.

Reply:
@ckuduvalli I am not a cron expert, but as  `/` is used to describe frequencies, I'd think that it is saying something like "every 30 hours". As the format is: `minutes, hours, day of month, mont, day of week` https://en.wikipedia.org/wiki/Cron#Overview

Reply:
so that's why I am asking since this specific example is a little bit confusing.

Reply:
Another thing, (I haven't seen the code yet), but you are mentioning, that you want to use, why don't you use the original corn: https://github.com/robfig/cron which is far more popular/stable on github and you are using a fork which seems to be not so popular and it's last commit is from Aug 2019?

Reply:
ringtail cron was an improvement over robfig. But I have changed it back to robfig since its a popular one. Will send out a code review by tomorrow. Testing out some stuff.

Reply:
@zroubalik please check the format now